readme.md

```diff
@@ -70,9 +70,9 @@ Now we can start the virtual machine by running the **up** command:
 
     $ vagrant up
 
-If you have added the domain to your **hosts** file, you can access the site via your web browser on port *8000*:
-  
-    http://project_name.dev:8000
+If you have added the domain to your **hosts** file, you can access the site via your web browser on port:
+
+    http://project_name.dev
```

Comment:
Hm. This is strange. 80 inside is supposed to be forwarded to 8000:
```
default: 80 => 8000 (adapter 1)
```
But I can't see it.